docs/TimelineGuide.md

```diff
@@ -211,10 +213,11 @@ This has two purposes.
 The first purpose is for tools, to autogenerate regular expression translations for triggers.
 
 The second purpose is for timelines at runtime.
-cactbot will use the `replaceSync` section to auto-replace anything inside a `sync /text`/ on a timeline line and the `replaceText` section to auto-replace anything inside the ability text.
+Cactbot will use the `replaceSync` section to auto-replace anything inside a `sync /text`/ on a timeline line,
+and the `replaceText` section to auto-replace anything inside the ability text.
 
-These do not match the entire line by default.
-So, some care is needed to make sure that replacements are not overzealous.
+These do not match the entire line (that is, they are non-greedy) by default.
```

Comment:
Maybe I am confused, but I don't follow what greedy means in this context.

Reply:
I'm not completely sure what I meant to say there myself. I think the update I made should better capture the sense of what the section is trying to say?